src/componentRenderer.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
```

Comment:
Note that only `src/componentRenderer.js` and `src/middleware.js` (plus tests) are new -- the rest is prettier